### PR TITLE
adding MedianComputable.java to stats

### PR DESCRIPTION
**Pull request by dvdhoz**
MedianComputable class

### CODE REVIEW

File: FIRO_TSEnsembles/src/main/java/hec/stats/MedianComputable.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+public class MedianComputable implements Computable {
+    @Override
+    public float compute(float[] values) {
+        //calculate the min of values
+        int n = values.length;
+        Arrays.sort(values);
+        if(n % 2 !=0) {
+            return values[n /2];
+        }
+        return (values[(n-1)/2] + values[n /2]) / 2;
```
Comment (HenryGeorgist) [inline FIRO_TSEnsembles/src/main/java/hec/stats/MedianComputable.java:14]: Do you have tests for this yet? I am suspicious that the rounding down is the default, not rounding up. i think you are hoping for rounding up?

Comment (HenryGeorgist) [reply FIRO_TSEnsembles/src/main/java/hec/stats/MedianComputable.java:14]: you may have to cast the result of n/2 to int for this to work.

File: FIRO_TSEnsembles/src/main/java/hec/stats/MedianComputable.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package hec.stats;
+
+import java.util.Arrays;
+
+public class MedianComputable implements Computable {
+    @Override
+    public float compute(float[] values) {
+        //calculate the min of values
```
Comment (HenryGeorgist) [inline FIRO_TSEnsembles/src/main/java/hec/stats/MedianComputable.java:8]: i dont think this comment applies anymore, i would clean it up. leaving it is confusing.

**Review by HenryGeorgist — CHANGES_REQUESTED** [review]

File: FIRO_TSEnsembles/src/main/java/hec/stats/MedianComputable.java (hunk 2 above)
Comment (dvdhoz) [reply FIRO_TSEnsembles/src/main/java/hec/stats/MedianComputable.java:8]: removed from code

File: FIRO_TSEnsembles/src/main/java/hec/stats/MedianComputable.java (hunk 1 above)
Comment (dvdhoz) [reply FIRO_TSEnsembles/src/main/java/hec/stats/MedianComputable.java:14]: tests added.  I believe its working as intended

File: FIRO_TSEnsembles/src/main/java/hec/stats/MedianComputable.java [hunk 3]
```diff
@@ -5,12 +5,11 @@
...
     public float compute(float[] values) {
-        //calculate the min of values
         int n = values.length;
         Arrays.sort(values);
         if(n % 2 !=0) {
-            return values[n /2];
+            return values[(n) /2];
         }
-        return (values[(n-1)/2] + values[n /2]) / 2;
+        return (values[(n)/2-1] + values[n /2]) / 2;
```
Comment (HenryGeorgist) [inline FIRO_TSEnsembles/src/main/java/hec/stats/MedianComputable.java:13]: great! that makes more sense.

**Review by HenryGeorgist — APPROVED** [review]

**Pull request merged**